02/ht/example/calendar/script.js

```diff
@@ -0,0 +1,105 @@
+function drawInteractiveCalendar(el) {
+  let titleCalendar = `<table class="title">
+                          <tr>
+                            <td><button class="last"><</button></td><td><span class="month"></span>/<span class="year"></span></td><td><button class="next">></button></td>
+                          </tr>
+                        </table>
+                        <div class="content"></div>
+                        <div class="content-notes"></div>`;
+  el.innerHTML = titleCalendar;
+
+  const titleElement = el.querySelector(".title");
+  const monthElement = el.querySelector(".month");
+  const yearElement = el.querySelector(".year");
+
+  const date = new Date();
+  drawCalendar(
+    date.getFullYear(),
+    date.getMonth(),
+    el.querySelector(".content")
+  );
+
+  const arrNotes = JSON.parse(localStorage.getItem(`${el.id}`)) || [];
+
+  el.querySelector(".content").addEventListener("click", function (e) {
+    const target = e.target;
+    if (Number(target.textContent)) {
+      const eventDate = prompt(`Заметка на ${target.textContent}`, "");
+      if (!eventDate) return;
+      const noteDate = `${target.textContent}.${
+        el.querySelector(".month").textContent
+      }.${el.querySelector(".year").textContent}: ${eventDate}`;
+      setTextLocal(noteDate, el, arrNotes);
+      showTextNote(el.querySelector(".content-notes"), arrNotes);
+    }
+  });
+
+  monthElement.textContent = date.getMonth() + 1;
+  yearElement.textContent = date.getFullYear();
+
+  titleElement.addEventListener("click", function (e) {
+    if (e.target.matches(".last")) {
+      date.setMonth(date.getMonth() - 1);
+    } else if (e.target.matches(".next")) {
+      date.setMonth(date.getMonth() + 1);
+    }
+    monthElement.textContent = date.getMonth() + 1;
+    yearElement.textContent = date.getFullYear();
+    drawCalendar(
+      date.getFullYear(),
+      date.getMonth(),
+      el.querySelector(".content")
+    );
+  });
+  el.addEventListener(
+    "load",
+    showTextNote(el.querySelector(".content-notes"), arrNotes)
+  );
+}
+
+function drawCalendar(year, month, htmlEl) {
```

Comment:
Уже есть похожий метод, добавь table так понятнее будет
```suggestion
function drawCalendarTable(year, month, htmlEl) {
```